angular/playground/projects/ng-entity-service/src/lib/types.ts

```diff
@@ -28,3 +30,28 @@ export type HttpConfig<Entity = any> = {
   url?: string;
   mapResponseFn?: (res: any) => Entity | Entity[];
 };
+
+interface StoreWrite {
+  /**
+   * Disables writing to the store
+   *
+   * You then have to manually write to the store.
+   * This is useful when pairing the NgEntityService with the PaginatorPlugin
+   */
+  skip?: boolean;
```

Comment:
Change `skip` to `write`

Reply:
Hmm this change is a bit complicated.

Flipping the meaning of the config would mean it have to be true by default so all the config parameters would now have to have a default value.
```ts
get<T>(id?: getIDType<S>, config?: HttpGetConfig<T> = { write: true })
// etc
```

I guess undefined could also mean true. But I think that makes it an unintuitive setting.

Reply:
So `skipWrite: boolean`